ibis/backends/tests/test_array.py

```diff
@@ -448,23 +448,27 @@ def test_array_slice(backend, start, stop):
         param({"a": [[1, 2], [4]]}, {"a": [[2, 3], [5]]}, id="no_nulls"),
     ],
 )
+@pytest.mark.parametrize(
+    "func,result_name",
+    [
+        (lambda x: x + 1, "ArrayMap(a, Add(x, 1))"),
+        (functools.partial(lambda x, y: x + y, y=1), "ArrayMap(a, Add(x, 1))"),
+        (ibis._ + 1, "ArrayMap(a, Add(_, 1))"),
+    ],
+)
```

Comment:
```suggestion
@pytest.mark.parametrize(
    "func",
    [
        lambda x: x + 1,
        functools.partial(lambda x, y: x + y, y=1),
        ibis._ + 1,
    ],
)
```

Generated names are an implementation detail and their specific value isn't guaranteed to be stable across any version of ibis.